src/styles/header.scss

```diff
@@ -0,0 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
+
+  &__logo-link {
+    display: block;
+    height: 40px;
+    width: 40px;
+  }
+
+  &__menu {
+    display: flex;
+
+    &-link {
+      position: relative;
+      display: flex;
+      align-items: center;
+      height: 60px;
+      text-transform: uppercase;
+      font-size: 12px;
+      line-height: 16px;
+      font-weight: 500;
+
+      &--is-active {
+        color: #00acdc;
+
+        &::after {
+          content: '';
+          position: absolute;
+          width: 100%;
+          height: 4px;
+          bottom: 0;
+          left: 0;
+          background-color: #00acdc;
+          border-radius: 2px;
+        }
+      }
+    }
+
+    &-item {
+      color: rgba(6, 11, 53, 1);
+
+      &:not(:last-child) {
+        margin-right: 20px;
+      }
```

Comment:
The `margin-right` value on `.header__menu-item:not(:last-child)` should use a spacing variable if one is available in the project for consistency.